src/Controller/Async/Stack.php

```diff
@@ -20,10 +21,20 @@ protected function addRoutes(ControllerCollection $c)
     {
         $c->post('/stack/add', 'add')
             ->assert('filename', '.*')
```

Comment:
This is probably a derp on my part from last time I modified this route. I don't think the assert is needed here anymore.